.github/workflows/ubuntu.yml

```diff
@@ -35,10 +35,13 @@ jobs:
                 -DNGX_OTEL_DEV=ON ..
           make -j $(nproc)
       - name: Download otelcol
+        env:
```

Comment:
I don't think this section should be used for script's local variables.

Reply:
Yes, It's not needed anymore.  Removed.